foxtrot-core/src/main/java/com/google/common/base/Stopwatch.java

```diff
@@ -0,0 +1,238 @@
+/*
+ * Copyright (C) 2008 The Guava Authors
```

Comment:
@r0goyal  This is because of hbase-server/hbase-client dependency on age old guava (12.0) which had a public constructor to Stopwatch. Dropwizard 0.9.2 required minimum 18.0. This breaks hell lot of stuff with hbase. There is no other way to maintain compatibility other than to make STopwatch compatible again with hbase and still not break dropowizard.